lib/Companies.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var Companies = function () {
+  function Companies(client) {
+    _classCallCheck(this, Companies);
+
+    this.__basePath = '/companies';
+    this.__client = client;
+  }
+
+  _createClass(Companies, [{
+    key: 'get',
+    value: function get(id) {
+      return this.__client.request(this.__urlFor('/' + id));
+    }
+  }, {
+    key: 'search',
+    value: function search(query) {
+      return this.__client.request(this.__urlFor(), { query: query });
+    }
+  }, {
+    key: '__urlFor',
+    value: function __urlFor() {
+      var path = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : '';
+
+      return this.__basePath + path;
+    }
+  }]);
+
+  return Companies;
+}();
+
+exports.default = Companies;
```

Comment:
Just haven't got around to setting it up as a npm module or adding `prepare` to package.json, which, theoretically, should allow a git package to be prepared in a similar fashion to how it would be done during the publishing (npm module) process.